diff --git a/api/peer_connection_interface.h b/api/peer_connection_interface.h
index 0ae47b2a2f..308fd06ca6 100644
--- a/api/peer_connection_interface.h
+++ b/api/peer_connection_interface.h
@@ -661,7 +661,7 @@ class RTC_EXPORT PeerConnectionInterface : public rtc::RefCountInterface {
     std::string turn_logging_id;
 
     // Added to be able to control rollout of this feature.
-    bool enable_implicit_rollback = false;
+    bool enable_implicit_rollback = true;
 
     // Whether network condition based codec switching is allowed.
     absl::optional<bool> allow_codec_switching;
